### ISSUE TITLE
calendar component not accessible

### ISSUE THREAD

**Issue by paciellogroup** [mozbrick/brick#53]
the calendar component (http://mozilla.github.io/brick/demos/calendar/index.html) is an interactive data table, but is not marked up as one and therefore the relationships between the data and headers is not conveyed (amongst other things) Suggest looking at the ARIA date picker design pattern http://www.w3.org/WAI/PF/aria-practices/#datepicker for info on making it understandable/usable accessibility wise

**Comment by potch** [comment]
Thanks for the pointer to the design pattern! We're looking to bake in full ARIA goodness before we reach 1.0 stable, and this will go a long way toward that.

**Comment by dotch** [comment]
will be tracked in the indiviudal repo at
https://github.com/mozbrick/brick-calendar/issues/4
